### DISCUSSION TITLE
HP ScanJet Pro 4500 fn1 - WORKS with airscan-driver

### DISCUSSION THREAD

**Discussion by wszoecs** [alexpevzner/sane-airscan#142] [Show and tell]
Hi,
I just got this thingie - which is a pure (networked) scanner (not an MFD) - and the hplip stuff just can't handle it.
The description how to compile this driver worked on Centos-8 - just via cut+paste - excellent :-)

The scanner gets detected:

$ scanimage -L
device `airscan:e0:HP ScanJet Pro 4500 fn1[AC781C]' is a eSCL HP ScanJet Pro 4500 fn1[AC781C] ip=10.20.30.88

...sure enough - xsane can handle it too.

COOL!

**Comment by alexpevzner** [comment]
Thanks!

I can add CentOS binary package, if somebody needs it.
